centreon/www/api/class/centreon_configuration_contactgroup.class.php

```diff
@@ -111,23 +115,25 @@ public function getList()
         }
 
         # get Ldap contactgroups
-        $ldapCgs = array();
-        if (isset($this->arguments['page_limit']) && isset($this->arguments['page'])) {
-            $maxItem = $this->arguments['page_limit'] * $this->arguments['page'];
-            if ($aclCgs['total'] <= $maxItem) {
+        if ($this->arguments['type'] != 'local') {
```

Comment:
Add a comment to explain please.
```suggestion
        if ($this->arguments['type'] !== 'local') {
```